src/content/documentation/develop/developing-extensions-for-firefox-for-android.md

```diff
@@ -83,16 +120,147 @@ If you are using a device:
   Where the hex string is your device’s (or emulator’s) code. This means adb has found your device (or emulator).
 
 {% endcapture %}
-{% include modules/column-w-toc.liquid
+{% include modules/one-column.liquid
   id: "set-up-your-computer-and-android-emulator-or-device"
-  content: content_with_toc
+  content: content
+%}
+
+<!-- END: Single Column Body Module -->
+
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## Check for Firefox for Android compatibility
+::: note alert
+This section only applies to Firefox for Android < 68 (Fennec)
+:::
```

Comment:
It looks that most of this content was already part of this document and we are mainly adding the note alert about the section being related to Fennec, but it looks as added in this pull request.

My guess is that we didn't remove the existing content parts and that it is still below and everything after line 260 should be removed, would you mind to check that and update the PR to remove all the duplicated sections?

Reply:
Haha you are asking for the hard thing. :) I really struggled with tracking changes in the markdown file yesterday, so I started a [Google doc](https://docs.google.com/document/d/1jxytLShEa5zWcqmiRStDnJr0mMi5ouAtKA1JgHi9JUk/edit#) where we can stage the edits and then I will  update the PR.

Reply:
Sorry @rpl, I commented earlier and then deleted the comment. I don't quite feel comfortable removing the Fennec info yet but maybe it makes more sense to move it to its own page.